### PR TITLE
Fix paginating

### PR DESCRIPTION
**Pull request by kiela**
This should fix paginating issue described in #94.

### CODE REVIEW

**Comment by brodock** [comment]
:+1:

File: lib/resque/server/views/status.erb [hunk 1]
```diff
@@ -1,18 +1,20 @@
 <%= status_view :status_styles, :layout => false %>
 
-<h1 class='wi'>Statuses: <%= @status.uuid %>/<%= @status.name %></h1>
+<h1 class='wi'>Status <%= @status.uuid %></h1>
 <p class='intro'>Viewing a specific job created with Resque::Plugins::Status. <a href="<%= u(:statuses) %>">Return to the list of statuses</a></p>
 
 <div class="status-holder" rel="<%= @status.status %>" id="status_<%= @status.uuid %>">
+  <h2>Overwiev</h2>
```
Comment (quirkey) [inline lib/resque/server/views/status.erb]: sp

**Comment by quirkey** [comment]
Looks good, will merge if you fix the spelling.

**Comment by brodock** [comment]
:shipit: ?

**Comment by quirkey** [comment]
Thanks

**Pull request merged**